DWIConvert/DWIConvert.cxx

```diff
@@ -67,6 +67,9 @@ int main(int argc, char *argv[])
     std::cout << "======= DWI Convert Public Lib Ctest =========" << std::endl;
     DWIConvert dWIConvert;
 
+    if (conversionMode == "FSLToNrrd" && (!fslNIFTIFile.empty()))
```

Comment:
Please add ITK style and use braces around the body of the if statement.

Reply:
alright, done.